src/Microsoft.DotNet.Helix/Sdk/tools/xharness-runner/xharness-event-processor.py

```
@@ -125,8 +125,8 @@ def analyze_operation(command: str, platform: str, device: str, is_device: bool,
 
     global retry, reboot, android_connectivity_verified
 
-    # Apps crashing can be infra failures, retry except on Apple devices where retries can be costly due to small queue size
-    if exit_code == 80 and not (platform == "apple" and is_device): # APP_CRASH
+    # Apps crashing can be infra failures too (this is a gray zone)
+    if exit_code == 80: # APP_CRASH
```

This logic is going to hide intermittent crashes. Is that correct?

---

Yes, this is a good point. Let's start distinguishing TCP-caused failures and let the other blow up.

(there have been 12 `APP_CRASH` results in the last 3 days btw)